apps/src/StudioApp.js

```diff
@@ -995,8 +1004,15 @@ StudioApp.prototype.onMouseMoveVizResizeBar = function (event) {
   visualizationColumn.style.maxWidth = (newVizWidth + vizSideBorderWidth) + 'px';
   visualization.style.maxWidth = newVizWidthString;
   visualization.style.maxHeight = newVizHeightString;
-  applyTransformScaleToChildren(visualization,
-      'scale(' + (newVizWidth / this.nativeVizWidth) + ')');
+
+  // We don't get the benefits of our responsive styling, so set height
+  // explicitly
+  if (!utils.browserSupportsCssMedia()) {
+    visualization.style.height = newVizHeightString;
```

Comment:
OK. My guess is that you only needed to set the height because the default size is 400 x 400 and the applab-specified size is 320 x 450 (w x h). so the width was already "big enough". So the concern would be that if applab ever specified a level with width > 400. Seems unlikely, but might be worth just setting width now as it could save us having to debug in IE9 to discover that this is the problem if we ever do make a wider applab level.